README.md

```diff
@@ -100,6 +100,21 @@ put(chan2, 42);
 const msg = await alts(chan1, chan2); // will receive 42
 ```
 
+### `select(Map<*, channel>|Set<channel>|Array<channel>|Object<string, channel>)` -> `Promise`
```

Comment:
awesome! could you just bump the number of exposed methods in API section intro.